xknx/io/udp_client.py

```diff
@@ -57,7 +57,7 @@ def error_received(self, exc):
 
         def connection_lost(self, exc):
             """Log error. Callback for connection lost."""
-            if hasattr(self, "xknx"):
+            if hasattr(self, "xknx") and exc is not None:
```

Comment:
if we test for exc not None we should at least log it too...
or just remove the whole if and `pass`

Reply:
Will add it, good point.